src/vue/modules/fees/components/fees-table.vue

```diff
@@ -60,29 +58,31 @@
               {{ { value: fee.upperBound, currency: fee.asset } | formatMoney }}
             </td>
           </tr>
+          <tr
+            :key="'no-data'"
+            v-if="!fees.length"
+          >
+            <td
+              class="fees-table__cell--align-center"
```

Comment:
1. invalid usage of BEM modifier. The modifier should be used with the element it modifies, i.g.: `fees-table__cell fees-table__cell--align-center`
2. invalid name, actually used for the row of with no-data meaning, but you named it "too general". if you name it like "fees-table__empty-list-placeholder" you can omit using of the BEM modifier